code/__DEFINES/lights.dm

```diff
@@ -1,5 +1,5 @@
 ///How much power emergency lights will consume per tick
-#define LIGHT_EMERGENCY_POWER_USE 0.2
+#define LIGHT_EMERGENCY_POWER_USE (0.2 KILO WATTS)
```

Comment:
Make sure you also scale it down to account for it getting multiplied by seconds_per_tick (which is 2 for machines).

Reply:
`LIGHT_EMERGENCY_POWER_USE & seconds_per_tick` gets passed on to `use_emergency_power()` proc so our scale is already applied for us

Reply:
Yes it does use as much power as the define says now, but emergency lights only consumed 100 watts due to consuming 200J per tick, which is per 2 seconds. So we halve the value to keep relative power usage on par with before the changes.

Reply:
Value has been halved